taskcluster/tc-build-utils.sh

```diff
@@ -312,7 +370,13 @@ do_nuget_build()
 
   cd ${DS_DSDIR}/native_client/dotnet
 
-  cp ${DS_TFDIR}/bazel-bin/native_client/libdeepspeech.so nupkg/build
+  # We copy the generated SO files into the Nuget runtime dirs
+
+  mkdir -p nupkg/runtimes/win-x64/native/  
+  cp ${DS_TFDIR}/bazel-bin/native_client/libdeepspeech.so nupkg/runtimes/win-x64/native/    
+  
+  mkdir -p nupkg/runtimes/win-arm64/native/
+  cp ${DS_TFDIR}/bazel-bin/native_client/libdeepspeech.so nupkg/runtimes/win-arm64/native/
```

Comment:
to me this reads as you are copying the same `.so` from above `x64` arch, to an `arm64` arch directory: `libdeepspeech.so` is platform specific, i dont see how this can go well

Reply:
maybe it is leftover that should be deleted? it feels like this is what your repackage functions are now doing (and with the comments I left, they should be able to do it correctly)